brianlong/www.validators.app
# 184866344 - remove ids from api json output

#### What's this PR do?
Removing id information occurrences from api json output. Changes impact stake pools, stake accounts and sol prices.

#### How should this be manually tested?
- ping `api/v1/stake-pools/:network` using both json and csv formats and check for any errors
- ping `api/v1/stake-accounts/:network` using both json and csv formats and check for any errors
- ping `api/v1/sol-prices` using both json and csv formats and check for any errors
- for stake accounts please verify the Stake Pools html page, since it is utilizing the stake account api endpoint
- enter the Sol Prices page and make it loads fully correct. 

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184866344)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
